hathorlib/scripts.py

```diff
@@ -139,7 +140,24 @@ def pushData(self, data: Union[int, bytes]) -> None:
             self.data += (bytes([Opcode.OP_PUSHDATA1]) + bytes([len(data)]) + data)
 
 
-class P2PKH:
+class BaseScript(ABC):
```

Comment:
Getting inspiration from `hathor.transaction.scripts.BaseScript` in hathor-core, we should add the `get_address(...)`, `get_type(...)`, and `get_script(...)` methods.

Reply:
I added only `get_type` and `get_script`. `get_address` method does not work for all the classes, e.g. `DataScript` does not have address.

Done here https://github.com/HathorNetwork/python-hathorlib/pull/6/commits/8f5a9764d41522728dde16540c509d65d07f6f64

Reply:
Great! Thanks!